ui/lib/ldap/addon/routes/roles.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+import type SecretMountPath from 'vault/services/secret-mount-path';
+import type StoreService from 'vault/services/store';
```

Comment:
We updated to use a separate `pagination` service in 1.19 so this is the main reason for needing to manually backport. 

This file should only reference `this.store.lazyPaginatedQuery`